docs/sources/tempo/configuration/_index.md

```diff
@@ -974,6 +974,13 @@ storage:
         # Default 1
         [blocklist_poll_tolerate_consecutive_errors: <int>]
 
+        # Polling will delete the index for a tenant if no blocks are found to
+        # exist.  The empty_tenant_deletion_age configuration is used to tune how
+        # quickly the poller will also delete any remaining backend objects found in the
+        # tenant path.
+        # Default: 20m; four times the blocklist_poll configuration above.
+        [empty_tenant_deletion_age: <duration>]
```

Comment:
20m seems quick, and in the range of normal block operations (maybe small network blips, and some compactions can take over an hour).  To start, what do you think about something more on the order of 2h to 24h?

Reply:
How about 12h default value?

Reply:
I've pushed a commit for the 12h default.

Reply:
Yep, 12h sounds good.  Just thinking about if this could interact with new tenants with partially written files, or other normal operations.